ledger-core/conveyor/conveyor.go

```diff
@@ -103,6 +103,10 @@ type PulseConveyor struct {
 	stoppingChan <-chan struct{}
 }
 
+func (p *PulseConveyor) GetDataManager() *PulseDataManager {
```

Comment:
It's not clear what is 'DataManager'. May be GetPulseDataManager?

Reply:
well, there should be more data in it soon, hence the name